{"name": "test_4", "status": "broken", "statusDetails": {"message": "TypeError: Failed expected string as 'msg' parameter, got 'AttributeError' instead.\nPerhaps you meant to use a mark?", "trace": "def test_4():\n    \n        try:\n    \n    # Open browser, maximazie window, get URL\n            obj_lib = Lib()\n            browser = obj_lib.open_browser()\n            obj_lib.page_load(browser)\n    \n    # Creating homepage & contact us page objects\n            obj_homepage = Homepage(browser)\n    \n    # Wait for dresses tab and click on it\n            obj_lib.wait_for_element(browser, obj_homepage.dresses_menu_xpath)\n>           browser.find_element(*obj_homepage.dresses_menu_xpath).click()\nE           AttributeError: 'NoneType' object has no attribute 'find_element'\n\nTest\\test_dresses_data.py:29: AttributeError\n\nDuring handling of the above exception, another exception occurred:\n\n    def test_4():\n    \n        try:\n    \n    # Open browser, maximazie window, get URL\n            obj_lib = Lib()\n            browser = obj_lib.open_browser()\n            obj_lib.page_load(browser)\n    \n    # Creating homepage & contact us page objects\n            obj_homepage = Homepage(browser)\n    \n    # Wait for dresses tab and click on it\n            obj_lib.wait_for_element(browser, obj_homepage.dresses_menu_xpath)\n            browser.find_element(*obj_homepage.dresses_menu_xpath).click()\n    \n    # Get products result\n            prod_result = obj_homepage.get_product_name_price(browser)\n    \n    # Open txt file write result\n            obj_lib.write_to_file(prod_result)\n    \n            print(\"TEST_4: PASS\")\n    \n        except Exception as error:\n            obj_lib.save_screenshot(browser)\n>           pytest.fail(error)\n\nTest\\test_dresses_data.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <[AttributeError(\"'Failed' object has no attribute 'msg'\") raised in repr()] Failed object at 0x3d97d48>\nmsg = AttributeError(\"'NoneType' object has no attribute 'find_element'\"), pytrace = True\n\n    def __init__(self, msg: Optional[str] = None, pytrace: bool = True) -> None:\n        if msg is not None and not isinstance(msg, str):\n            error_msg = (\n                \"{} expected string as 'msg' parameter, got '{}' instead.\\n\"\n                \"Perhaps you meant to use a mark?\"\n            )\n>           raise TypeError(error_msg.format(type(self).__name__, type(msg).__name__))\nE           TypeError: Failed expected string as 'msg' parameter, got 'AttributeError' instead.\nE           Perhaps you meant to use a mark?\n\n..\\..\\..\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\_pytest\\outcomes.py:40: TypeError"}, "attachments": [{"name": "stdout", "source": "ccc4ebeb-5a2c-4cbf-8730-413877d7efe4-attachment.txt", "type": "text/plain"}], "start": 1591706115683, "stop": 1591706119560, "uuid": "acf03d26-422a-4fe2-8367-5d41db982d95", "historyId": "603507f6fe7f693bf2ee01c0a6b6d552", "testCaseId": "16dc294522e713159c19f574690ef105", "fullName": "Test.test_dresses_data#test_4", "labels": [{"name": "parentSuite", "value": "Test"}, {"name": "suite", "value": "test_dresses_data"}, {"name": "host", "value": "WIN-GKTDII34UKV"}, {"name": "thread", "value": "4068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test.test_dresses_data"}]}